src/dodal/devices/util/lookup_tables_apple2.py

```diff
@@ -408,12 +329,61 @@ def get_motor_from_energy(self, energy: float, pol: Pol) -> tuple[float, float]:
             (gap, phase) motor positions.
         """
         if self.available_pol == []:
-            self.update_lookuptables()
+            self.update_lookup_table()
+        poly = get_poly(lookup_table=self.lut, energy=energy, pol=pol)
+        return poly(energy)
+
+
+TAbstractEnergyMotorLookup = TypeVar(
+    "TAbstractEnergyMotorLookup", bound=AbstractEnergyMotorLookup
+)
+
+
+class FileReadingEnergyMotorLookup(AbstractEnergyMotorLookup):
```

Comment:
Should: We not realy doing file reading here, we getting the data from DAQ server.
```suggestion
class EnergyMotorLookupConfigServer(AbstractEnergyMotorLookup):
```